CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ include(CMakePackageConfigHelpers)
 #   backwards-compatible changes or bug fixes.
 project(HindsightCAN
   LANGUAGES C
-  VERSION 1.1.3) # <-- Change when you make commits to master; see above
+  VERSION 1.1.4) # <-- Change when you make commits to master; see above
```

Comment:
I'd say bump the minor version, not the patch, since this is a feature addition not a bugfix. So the new version should be 1.2.0.

Breaking changes should bump the major version, nonbreaking feature additions should bump the minor version, and bugfixes should bump the patch version.